packages/io/io/src/formats.js

```diff
@@ -113,22 +104,6 @@ const supportedFormats = {
   }
 }
 
-// handled input formats that can be converted to jscad code
-const conversionFormats = [
```

Comment:
conversionFormats is exported from `formats.js` and re-exported in `index.js` but does not exist. This throws an error when I try to run tests.